src/index.ts

```diff
@@ -1,32 +1,60 @@
-import { connect } from "nats";
+import { connect, NatsConnection } from "nats";
 import express, { json } from "express";
 import cors from "cors";
-
 import restToNatsBridge from "./middlewares/natsToRestBridge";
 import serveOpenAPIDocs from "./middlewares/openAPIDocs";
 import { HTTP_PORT, NATS_URL } from "./config";
 import errorHandlingMiddleware from "./middlewares/errorHandler";
 import authorizationMiddleware from "./middlewares/authorization";
+import { drain } from "./lib/nats/nats";
 
-async function init() {
-    const natsConnection = await connect({
-        servers: NATS_URL
-    });
-
-    const app = express();
-
-    app.use(json());
+let natsConnection: NatsConnection;
 
-    app.get("/docs", cors(), serveOpenAPIDocs(natsConnection));
-    app.use("/", cors(), authorizationMiddleware);
-    app.use("/api", cors(), restToNatsBridge(natsConnection));
-    app.use(errorHandlingMiddleware);
-
-    console.info(`[AIRLOCK] Connected to Nats ${natsConnection.getServer()}`);
+async function shutdown(
```

Comment:
I would move this function in the scope of the `init()` function so I don't have to pass the natsConnection everytime